GOVERNANCE.md

```diff
@@ -6,25 +6,25 @@ governance structure.
 
 To that end, this document describes the principles
 that guide the self-management of the project.
-By having them written down explicitly, and open to scrutiny,
+By having them written down explicitly and open to scrutiny,
 the entire community can read, apply, improve and adapt them as needed,
 with no central authority.
 
-Community members are asked to abide by the following principles:
+Community members are requested to abide by the following principles:
 
 1. **All contributions are welcome**,
    [no matter how small](https://github.com/kentcdodds/all-contributors).
    The tldr-pages project is a
    [do-ocracy](https://communitywiki.org/wiki/DoOcracy),
    so don't hesitate to get involved
    — we're happy to welcome you into the community!
-   Please take a look at [CONTRIBUTING.md](CONTRIBUTING.md) to get started.
+   Please take a look at the [CONTRIBUTING.md](CONTRIBUTING.md) file to get started.
 
 2. **All interactions must be respectful and cordial**.
-   Avoid making assumptions about the others' intentions,
-   and make your own intentions clear.
-   When in doubt, provide additional context, or ask for clarification.
-   Remember, it's very hard to convey meaning on a purely written medium,
+   Avoid making assumptions about the other's intentions,
+   and make your intentions clear.
+   When in doubt, provide additional context or ask for clarification.
+   Remember, its very hard to convey meaning in a purely written medium,
```

Comment:
```suggestion
   Remember, it's very hard to convey meaning in a purely written medium,
```

This was indeed a correct "it's", as it can be replaced by "It is".